docs/tutorial2023_unfolding/unfolding_exercise.md

```diff
@@ -0,0 +1,256 @@
+# Combine unfolding exercise
+
+## Getting started
+By now you should have a working setup of Combine v9 from the pre-tutorial exercise. If so then move onto the cloning of the parametric fitting exercise gitlab repo below. If not then you need to set up a CMSSW area and checkout the combine package:
+```shell
+cmsrel CMSSW_11_3_4
+cd CMSSW_11_3_4/src
+cmsenv
+git clone https://github.com/cms-analysis/HiggsAnalysis-CombinedLimit.git HiggsAnalysis/CombinedLimit
+cd HiggsAnalysis/CombinedLimit
+
+cd $CMSSW_BASE/src/HiggsAnalysis/CombinedLimit
+git fetch origin
+git checkout v9.1.0
+```
+We will also make use of another package, `CombineHarvester`, which contains some high-level tools for working with combine. The following command will download the repository and checkout just the parts of it we need for this exercise:
+```shell
+cd $CMSSW_BASE/src/
+bash <(curl -s https://raw.githubusercontent.com/cms-analysis/CombineHarvester/main/CombineTools/scripts/sparse-checkout-https.sh)
+```
+Now let's compile the CMSSW area:
+```shell
+scramv1 b clean; scramv1 b
+cmsenv
+```
+Finally, let's clone the working directory for this tutorial which contains all of the inputs and scripts needed to run the unfolding fitting exercise:
+```shell
+cd $CMSSW_BASE/src/
+git clone https://gitlab.cern.ch/cms-analysis/general/combine-unfolding-tutorial-2023.git
+cd combine-unfolding-tutorial-2023
+```
+
+## Exercise outline
+
+The hands-on exercise is split into seven parts: 
+
+1) Counting experiment
```

Comment:
It  wonder if it might be more clear to change this name from something like "counting experiment" to "simplified analysis strategy" or "simplified unfolding"?

And similarly, maybe changing "Shape analysis with control regions" to something like "Extending the analysis with other variables and control regions"